deploy/monitor/grafana_dashboard.yaml

```diff
@@ -0,0 +1,494 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: mobile-developer-console-operator
+  labels:
+    monitoring-key: middleware
+    prometheus: application-monitoring
+spec:
+  selector:
+    matchLabels:
+      name: mobile-developer-console-operator
+  json: |
+    {
+      "__inputs": [
+          {
+            "name": "DS_PROMETHEUS",
+            "label": "Prometheus",
+            "description": "",
+            "type": "datasource",
+            "pluginId": "prometheus",
+            "pluginName": "Prometheus"
+          }
+        ],
```

Comment:
I don't think the source is required, we can remove it and the default one will be picked up.

Reply:
I wasn't sure about this one. Could you check the new commit here and see if I've removed enough or too much?